qemu/tests/lvm_with_read_only.py

```diff
@@ -0,0 +1,45 @@
+import time
+
+from avocado.utils import process
+
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Boot guest after disable ept/npt:
+    1) Create lvm with read-only permission
+    2) Boot up guest with lvm
+    3) Unplug the disk
+    4) Hotplug blockdev node with auto-read-only
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    def create_lvm():
+        devname = process.getoutput(params["get_devname_command"])
+        pvcreate_command = (params["pvcreate_command"] % devname)
+        process.system(pvcreate_command)
+        vgcreate_command = (params["vgcreate_command"] % devname)
+        process.system(vgcreate_command)
+        process.system(params["lvcreate_command"])
+
+    create_lvm()
+    params["start_vm"] = 'yes'
+    env_process.preprocess_vm(test, params, env, params["main_vm"])
+    vm = env.get_vm(params['main_vm'])
+    session = vm.wait_for_login()
+    qmp_port = vm.monitor
+
+    try:
+        qmp_port.send_args_cmd(params["unplug_qmp_command0"])
+        time.sleep(5)
+        qmp_port.send_args_cmd(params["unplug_qmp_command1"])
+        qmp_port.send_args_cmd(params["unplug_qmp_command2"])
+        qmp_port.send_args_cmd(params["plug_qmp_command"])
```

Comment:
Please use `simple_unplug` from `./virttest/qemu_devices/qcontainer.py` or you could use `BlockDevicesPlug.unplug_devs_serial` from `./test-providers.d/downloads/io-github-autotest-qemu/provider/block_devices_plug.py`

Reply:
Thanks for your guidance and updated the code. Could you please review this patch again?